parasol/solr/update.py

```diff
@@ -0,0 +1,47 @@
+import json
+from urllib.parse import urljoin
+
+
+class Update:
+    '''Class for managing the update handler functionality of the Solr API.'''
```

Comment:
Maybe just: "API client for Solr update functionality" ?  

Current wording is a bit vague ("managing update handler"). Similar for schema API object.